lib/core/presentation/pages/home/home_page.dart

```diff
@@ -50,7 +50,12 @@ class _HomePageState extends State<HomePage> {
           GestureDetector(
             behavior: HitTestBehavior.translucent,
             onTap: () {
-              AutoRouter.of(context).navigate(const ChatListRoute());
+              final authState = BlocProvider.of<AuthBloc>(context).state;
+              if (authState is AuthStateAuthenticated) {
```

Comment:
we can use state.maybeWhen(authenticated: () => doSth, orElse: showLogin)

Reply:
Thanks 👍